main.js

```diff
@@ -0,0 +1,32 @@
+let user = 'John Doe';
+console.log(user);
+let student = 'Lolita';
+console.log(student);
+user = student; //Lolita
+console.log(user);
+let test = 1;
+test++;
+test = test + '1'; //21
+console.log(test);
+test--; //11
```

Comment:
maybe the result should be 20?

Reply:
@lolitamalinovskaya

Reply:
It not important at all - another code has no problems to fix it, I will merge it and you can fix it in another branch (generally commit shouldn't have any comments ).